infrastructure/resource_group_level/main.bicep

```diff
@@ -86,6 +89,28 @@ resource containerRegistry 'Microsoft.ContainerRegistry/registries@2023-01-01-pr
   }
 }
 
+resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2022-10-01' = {
```

Comment:
It may be appropriate to tie all Application instances with a workspace - this is the recommended way now.
However, I'll just link the ones that are currently wired up in `Dev` for now.

Reply:
sounds good